src/HTMLConverter.php

```diff
@@ -111,4 +134,40 @@ public function td(string $fieldname_attribute_name): self
 
         return $clone;
     }
+
+    /**
+     * Create a DOMElement representing a single record of data
+     *
+     * @param string $rowTagName
+     * @param string $elementTagName
+     * @param array $record
+     * @param DOMDocument $doc
+     *
+     * @return DOMElement
+     */
+    private function createRecordRow(string $rowTagName, string $elementTagName, array $record, DOMDocument $doc)
```

Comment:
you can typehint the return type directly thanks PHP7 😉

Reply:
added the PHP7 hint but not removed the phpdock hint as I was not sure if you implied that or not?